src/ru/sberbank/javaSchool/transport/TransportBuilder.java

```diff
@@ -0,0 +1,45 @@
+package ru.sberbank.javaSchool.transport;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
+public abstract class TransportBuilder<T extends Transport> {
+    protected String vin;
+    protected GregorianCalendar dateOfIssue = new GregorianCalendar(1886, Calendar.JANUARY, 29);
```

Comment:
the best way is using new date types in java https://www.baeldung.com/java-8-date-time-intro